src/ot_orchestration/operators/gcs.py

```diff
@@ -34,15 +35,14 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(*args, **kwargs)
-        # try to get upload url from yaml_file
         self.project_id = project_id
         self.dst = GCSPath(dst)
         self.src = src
 
         self.bucket_name, self.path = self.dst.split()
 
     def execute(self, context) -> None:
-        """Upload the configurations to GCS."""
+        """Execute the Operator."""
```

Comment:
This kind of docstring is very trivial. I wonder if we really need them.

Reply:
We don't, I agree completely. But pre-commit complains if I remove it and I didn't find a way to add ignores by method name. :cry:

Reply:
Let's keep in mind that and try to resolve in future.